Websites/webkit.org/wp-content/themes/webkit/style.css

```diff
@@ -2549,6 +2554,7 @@ p .search-term {
         padding: 0 3rem 3rem 0;
     }
 
+    #site-nav .search-input,
     .search-input {
```

Comment:
why do we need to add the `#site-nav`?  seems unnecessary

Reply:
There is a generic definition of `.search-input` used everywhere as the search input can be used in dark and light parts of the page regardless of dark and light mode preferences. 

The issue this patch fixes only occurs in the site navigation in the header, but only when the header uses desktop layout. At smaller viewport sizes the menu design changes and makes use of the originally defined default. This contextual override is necessary.

I believe it is scoped appropriately.

Reply:
This specific override, resets the design to back to the defaults in smaller viewport layouts.